docker-compose.yml

```diff
@@ -205,6 +205,29 @@ services:
       KAFKA_LDAP_USER_NAME_ATTRIBUTE: uid
       KAFKA_LDAP_USER_OBJECT_CLASS: inetOrgPerson
 
+      # ConfluentTelemetryReporter: Disable
+      KAFKA_CONFLUENT_TELEMETRY_REPORTER_ENABLE: 'false'
```

Comment:
@rnpridgeon 

1. Why is telemetry reporter disabled?
2. Should it be enabled in cp-demo?  (cc: @sdanduConf )

Reply:
This was a work around because the internal exporter topic had a default replication factor of 3 and attempts to override it did not work. I think this has been fixed and it can be removed

Reply:
@rnpridgeon cool, so will you be flipping this to `true`?

Reply:
I'll just remove it entirely